packages/vite/src/node/logger.ts

```diff
@@ -189,14 +199,24 @@ function printServerUrls(
             // Node >= v18
             (typeof detail.family === 'number' && detail.family === 4))
       )
-      .map((detail) => {
-        const type = detail.address.includes('127.0.0.1')
-          ? 'Local:   '
-          : 'Network: '
+      .forEach((detail) => {
         const host = detail.address.replace('127.0.0.1', hostname.name)
         const url = `${protocol}://${host}:${colors.bold(port)}${base}`
-        return `  > ${type} ${colors.cyan(url)}`
+        const label = detail.address.includes('127.0.0.1') ? 'Local' : 'Network'
+
+        urls.push({ label, url: colors.cyan(url) })
       })
-      .forEach((msg) => info(msg))
   }
+
+  const length = urls.reduce(
+    (length, { label }) => Math.max(length, label.length),
+    0
+  )
+  urls.forEach(({ label, url: text }) => {
+    info(
+      `    ${colors.green('➜')}  ${colors.bold(label)}: ${' '.repeat(
```

Comment:
Love the design. I prefer 2 spaces here like in No.3 design since I think the extra 2 spaces don't provide special meaning here.

Reply:
I also prefer 2 spaces here. It is just a detail but aligning with the start of `VITE` feels nicer.